db/schema.rb

```diff
@@ -10,7 +10,8 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.2].define(version: 2025_02_28_161501) do
+
+ActiveRecord::Schema[7.2].define(version: 2025_03_03_152123) do
```

Comment:
Think it's added another line break here, can we move this back up to line 13 please